nservicebus/messaging/headers.md

```diff
@@ -42,17 +48,30 @@ Use the following code to generate or read the NServiceBus custom timestamp form
 ```cs
 const string Format = "yyyy-MM-dd HH:mm:ss:ffffff Z"; // Not ISO 8601
 
+#if-version [8,)
```

Comment:
The existing code does not have anything to do with either `DateTimeOffsetHelper` or `DateTimeExtensions`. There is no need for versioned snippets as the usage of `DateTime` or `DateTimeOffset` is unrelated to NServiceBus.

Reply:
This snippet should not be used where there is a reference to NServiceBus.